webapp/graphql/User.js

```diff
@@ -14,7 +14,7 @@ export default (i18n) => {
     ${userCountsFragment}
     ${locationAndBadgesFragment(lang)}
 
-    query User($id: ID!) {
+    query User($id: ID!, $followedByCount: Int, $followingCount: Int) {
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-04-07T12:35:50Z" title="Tuesday, April 7th 2020, 2:35:50 pm +02:00">Apr 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -14,7 +14,7 @@ export default (i18n) => {
     ${userCountsFragment}
     ${locationAndBadgesFragment(lang)}
 
-    query User($id: ID!) {
+    query User($id: ID!, $followedByCount: Int, $followingCount: Int) {
```


What happens if we don't pass the counts into the `User` query? Do we need to add a fallback? Have to admit, I didn't check the backend...

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-09T20:34:28Z" title="Thursday, April 9th 2020, 10:34:28 pm +02:00">Apr 9, 2020</time>_

---


Backend returns all entities, when `first` parameter is not set or `null`.

https://github.com/Human-Connection/Human-Connection/pull/3293/files/72dfeb3983689737afbdafeda1ec1a5862da3754#diff-399a892519b3cfbb5e4649fc25b0be0cR28